src/less/availability.less

```diff
@@ -1,10 +1,16 @@
 @import './variables.less';
 
-.rbc-availability {
+.rbc-availability,
+.rbc-availability-overlap {
   background-color: @availability-bg;
   border: @availability-border;
   border-radius: @availability-border-radius;
   color: @availability-color;
   padding: @availability-padding;
   width: @availability-width;
```

Comment:
Can we move these into a separate `.rbc-availability` selector that doesn't also include `.rbc-availability-overlap`? It's a little confusing that `width` and `padding` are here inside of a selector that applies to both overlap and non-overlap availabilities but the values here will never take effect for overlaps because they get immediately overridden by the next CSS rule below this.